### PR TITLE
PR and Issues Templates

### PR DESCRIPTION
**Pull request by aaishikasb**
# Added the following:

- [x] Pull Request Template
- [x]  #Issues Template

### CODE REVIEW

**Comment by aaishikasb** [comment]
#40

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+## Bug Reports
+Please replace this line with a brief summary of your issue AND if reporting a build issue include the link.
+
+[ ] Didn't run?
+[ ] Permission Issues?
+[ ] Broken Links
+[ ] Other Issues? (Add Here)
+
+## Want to help solve this Issue?
```
Comment (kaustubhgupta) [inline .github/issue_template.md:13]: Replace this section with an additional resources section where any links, references are can be put by the user

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Make this change and you are good to go!

File: .github/issue_template.md (hunk 1 above)
Comment (aaishikasb) [reply .github/issue_template.md:13]: On it!

**Comment by aaishikasb** [comment]
Hey, since the PR was merged and it was also a valid issue cited by you, why did you mark it as invalid? @kaustubhgupta

**Comment by kaustubhgupta** [comment]
@aaishikasb because you made two PRs and closed this one. Your other PR has been approved: https://github.com/kaustubhgupta/Hacktoberfest2020_/pull/44

**Pull request closed without merging**